File: 20251105_Contenido_info.js
/*Cómo crear una matriz en JavaScript:

Para empezar, podemos crear una función personalizada que acepta dos parámetros: el número de filas y el número de columnas. Creamos una variable para almacenar el resultado. Creamos un ciclo for con el número de filas y asignamos el resultado al variable que creamos. Adentro de ese ciclo for, creamos otro ciclo for para poblar la matriz con ceros y asignar el resultado a la variable. Retornamos la matriz. Después simplemente imprimimos.*/

function crea_matriz(filas, columnas) {
  const matriz = [];
  for (let i = 0; i < filas; i++) {
    matriz[i] = [];
    for (let j = 0; j < columnas; j++){
        matriz[i][j] = 0;
      }
  }
  return matriz;

}

console.log(crea_matriz(2, 3))

File: 20251105_Contenido_info.py
"""
Video: youtube.com/watch?v=IySyW49o17Y

Cómo crear una matriz en Python:

Para empezar, podemos crear una función personalizada que acepta dos parámetros: el número de filas y el número de columnas. Creamos un ciclo for con el número de columnas como rango en la forma de una lista adentro de otra lista que contiene un ciclo for con el número de filas como rango y lo asignamos a una variable. Retornamos la matriz. Después simplemente imprimimos.
"""

def crea_matriz(filas, columnas):
    matriz = [[0 for columna in range(columnas)] for fila in range(filas)] 
    return matriz

print(crea_matriz(2, 3))
